packages/lab/src/QueryBuilder/Rule/Operator/Operator.js

```diff
@@ -33,6 +33,7 @@ const Operator = ({ id, combinator, attribute, operator }) => {
       placeholder={labels.rule.operator.placeholder}
       values={values}
       disabled={values.length === 0}
+      readOnly={readOnly}
```

Comment:
Not really related to the QueryBuilder, but I think it would be nice to be able to select the values from `readOnly` dropdowns. Currently we can't because of the `userSelect: "none"`.

We could enable `userSelect` in `headerReadOnly`

![image](https://user-images.githubusercontent.com/638946/184341361-f63d66d4-0fdc-4349-b8c7-b7caf602141a.png)

Just an idea 😃

Reply:
Ah, that "kind of" came up when doing this work, but I'll check with the DS team to see what they think about it.

Reply:
Updated the PR with this change since it can be considered part of this work.

Reply:
awesome!